### PR TITLE
Feature: Allow a data only export/import cycle

### PR DESCRIPTION
**Pull request by stumpylog**
## Proposed change

Specifically to enable database upgrades with less work, this allows the export and import to operation on only data, not files.  It's a decent burden to export, remove all existing files, create a new database, then import again.

## Type of change

- [ ] Bug fix: non-breaking change which fixes an issue.
- [x] New feature / Enhancement: non-breaking change which adds functionality. _Please read the important note above._
- [ ] Breaking change: fix or feature that would cause existing functionality to not work as expected.
- [ ] Documentation only.
- [ ] Other. Please explain:

## Checklist:

- [ ] I have read & agree with the [contributing guidelines](https://github.com/paperless-ngx/paperless-ngx/blob/main/CONTRIBUTING.md).
- [ ] If applicable, I have included testing coverage for new code in this PR, for [backend](https://docs.paperless-ngx.com/development/#testing) and / or [front-end](https://docs.paperless-ngx.com/development/#testing-and-code-style) changes.
- [ ] If applicable, I have tested my code for new features & regressions on both mobile & desktop devices, using the latest version of major browsers.
- [ ] If applicable, I have checked that all tests pass, see [documentation](https://docs.paperless-ngx.com/development/#back-end-development).
- [ ] I have run all `pre-commit` hooks, see [documentation](https://docs.paperless-ngx.com/development/#code-formatting-with-pre-commit-hooks).
- [ ] I have made corresponding changes to the documentation as needed.
- [ ] I have checked my modifications for any breaking changes.

### CODE REVIEW

**Comment by stumpylog** [comment]
The missing lines weren't covered in dev either.  Maybe I'll check on them in a different pr

File: src/documents/management/commands/document_importer.py [hunk 1]
```diff
@@ -57,18 +57,20 @@ class Command(BaseCommand):
...
-    def __init__(self, *args, **kwargs):
-        BaseCommand.__init__(self, *args, **kwargs)
-        self.source = None
-        self.manifest = None
-        self.version = None
+        parser.add_argument(
+            "--data-only",
+            default=False,
+            action="store_true",
+            help="If set, only the database will be exported, not files",
```
Comment (shamoon) [inline src/documents/management/commands/document_importer.py:72]:
```suggestion
            help="If set, only the database will be imported, not files",
```

=)

**Review by shamoon — APPROVED** [review]
Did notice one tiny typo, otherwise I dont have much comment on the diff (seems like you included some other cleanup stuff?).

Overall works great, does seem really useful for db upgrades (inspiring me to do so myself). Thanks stumpy!

File: src/documents/management/commands/document_importer.py (hunk 1 above)
Comment (stumpylog) [reply src/documents/management/commands/document_importer.py:72]: Can tell where I copied it from...

**Comment by stumpylog** [comment]
I couldn't resist a little extra cleanup.  Plus it did make it easier to reason about which parts could be skipped

**Pull request merged**